os/board/rtl8720e/src/rtl8720e_boot.c

```diff
@@ -336,6 +336,9 @@ void board_initialize(void)
 #ifdef CONFIG_AMEBALITE_WIFI
 	wlan_initialize();
 #endif
+#if defined(CONFIG_BT_AP) && CONFIG_BT_AP
```

Comment:
Could you check whether proper header (it looks `os/board/rtl8720e/src/project/realtek_amebaLite_va0_example/inc/inc_km4/platform_autoconf.h`) is included well or not? I can't find that in header including.

Reply:
I have replaced CONFIG_BT_AP with CONFIG_AMEBALITE_BLE which is defined in config.h.